cluster/examples/kubernetes/ceph/cluster.yaml

```
@@ -37,6 +37,10 @@ spec:
   skipUpgradeChecks: false
   # Whether or not continue if PGs are not clean during an upgrade
   continueUpgradeAfterChecksEvenIfNotHealthy: false
+  # WaitTimeoutForHealthyOSD defines the time (in minutes) the operator would wait before an OSD can be stopped for upgrade or restart.
+  # It also defines the time the rook will wait for PGs to become active+clean after an OSD daemon is upgraded or restarted.
+  # The default wait timeout is 5 minutes.
+  # waitTimeoutForHealthyOSD: 10
```

How about uncommenting and setting the default?
```suggestion
  waitTimeoutForHealthyOSD: 5
```